### DISCUSSION TITLE
Navigation side panel and search panel are not working

### DISCUSSION THREAD

**Discussion by Tinhone** [community/community#103255] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I want to open navigation side panel and clicked my avater on the top bar, there is no response on the page.
当我想要打开导航侧方面板并点击顶栏上的头像时，页面没有任何响应。

When I want to open search panel and clicked search bar or type `/`, there is no response on the page. But I can use search function by clicking on the command palette button next to the search bar.
当我想要打开搜索面板并点击搜索栏或输入 `/` 时，页面没有任何响应。但是我可以通过点击搜索栏旁边的命令面板按钮来使用搜索功能。

According to this [MDN page](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement), personally, I think it's a compatibility issue.
根据此 [MDN 页面](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement)，我个人认为这是一个兼容性问题。

### Repro

1. https://github.com/
2. Click user's avatar on the top bar
3. The problem arises immediately

\--------
1. https://github.com/
2. 点击顶栏上的头像
3. 问题即刻出现

### Possible Solutions

Return the relevant code to the old version, not using [HTMLDialogElement related technologies](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement).
将相关代码退回旧版本，不使用 [HTMLDialogElement 相关技术](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement)。

### Other

Browser:
浏览器：
Mozilla Firefox v88.0

Log:
控制台日志：
```
Content Security Policy: 页面设置阻止读取位于 inline 的一项资源("script-src")。 [content.js:59:206](moz-extension://fe93d048-a1f1-4986-8549-51b5ca34379f/content.js)
Content Security Policy: 页面设置阻止读取位于 inline 的一项资源("script-src")。 [content.js:59:421](moz-extension://fe93d048-a1f1-4986-8549-51b5ca34379f/content.js)
Uncaught ReferenceError: HTMLDialogElement is not defined
    tC dialog_helper.js:22
[dialog_helper.js:22:12](https://github.githubassets.com/assets/node_modules/@primer/view-components/app/components/primer/dialog_helper.js)
    tC dialog_helper.js:22
Uncaught (in promise) ReferenceError: HTMLDialogElement is not defined
    expand qbsearch-input-element.ts:547
    handleExpand qbsearch-input-element.ts:534
    l bind.js:73
[qbsearch-input-element.ts:547:8](https://github.githubassets.com/assets/app/components/search/qbsearch-input-element.ts)
    expand qbsearch-input-element.ts:569
    handleExpand qbsearch-input-element.ts:534
    l bind.js:73
Uncaught ReferenceError: HTMLDialogElement is not defined
    tC dialog_helper.js:22
[dialog_helper.js:22:12](https://github.githubassets.com/assets/node_modules/@primer/view-components/app/components/primer/dialog_helper.js)
    tC dialog_helper.js:22
Uncaught (in promise) ReferenceError: HTMLDialogElement is not defined
    expand qbsearch-input-element.ts:547
    handleExpand qbsearch-input-element.ts:534
    l bind.js:73
[qbsearch-input-element.ts:547:8](https://github.githubassets.com/assets/app/components/search/qbsearch-input-element.ts)
    expand qbsearch-input-element.ts:569
    handleExpand qbsearch-input-element.ts:534
    l bind.js:73
Uncaught ReferenceError: HTMLDialogElement is not defined
    tC dialog_helper.js:22
[dialog_helper.js:22:12](https://github.githubassets.com/assets/node_modules/@primer/view-components/app/components/primer/dialog_helper.js)
    tC dialog_helper.js:22
Uncaught ReferenceError: HTMLDialogElement is not defined
    tC dialog_helper.js:22
[dialog_helper.js:22:12](https://github.githubassets.com/assets/node_modules/@primer/view-components/app/components/primer/dialog_helper.js)
Uncaught (in promise) ReferenceError: HTMLDialogElement is not defined
    expand qbsearch-input-element.ts:547
    handleExpand qbsearch-input-element.ts:534
    l bind.js:73
[qbsearch-input-element.ts:547:8](https://github.githubassets.com/assets/app/components/search/qbsearch-input-element.ts)
```

**Comment by imjxy6** [comment]
I also facing the same issue.

**Comment by ghost** [comment]
I'm surprised that the GitHub web devs didn't even bother trying to polyfill HTMLDialogElement.

**Comment by imjxy6** [comment]
Update: After Updating My Browser (Firefox), Everything Works Fine.

**Reply by ghost** [reply]
Yes, because new versions of Firefox support it. But it was too recently introduced for the GitHub devs to not polyfill it.
